wordpress-mobile/WordPress-Android
# Fix toolbar titles in PhotoPicker

Fixes #11754 

Fixes titles in the media picker.


~Note: The PR is targeting a release branch. If this will be the only PR targeting the release branch, I don't think it's worth releasing a new version. (cc @oguzkocer)~

To test:
1. Open the Gutenberg editor
2. Add an image block
3. Click on "Add Image"
4. Verify the title is "Choose photo"
5. Select an item
6. Verify the title is "Use this photo"
-------------------------
1. Open the Gutenberg editor
2. Add an video block
3. Click on "Add video"
4. Verify the title is "Choose video"
5. Select an item
6. Verify the title is "Use this video"
-------------------------
1. Open the Gutenberg editor
2. Add an text&media block
3. Click on "Add media"
4. Verify the title is "Choose media"
5. Select an item
6. Verify the title is "Use this media"
-------------------------
1. Open the Gutenberg editor
2. Add an gallery block
3. Click on "Add media"
4. Verify the title is "Choose photo"
5. Select an item
6. Verify the title is "1 selected"
-------------------------
PR submission checklist:
